sklearn/tree/_splitter.pyx

```diff
@@ -19,8 +19,8 @@ from libc.string cimport memcpy
 from libc.string cimport memset
 
 import numpy as np
-cimport numpy as np
-np.import_array()
+cimport numpy as cnp
+cnp.import_array()
```

Comment:
I do not think we need this import anymore.

```suggestion
```

Reply:
Yes, I did not realise the code you removed in https://github.com/scikit-learn/scikit-learn/pull/23273 was the only occurrence of `cnp`. Thanks for pointing it out.